book-examples/leptos/Cargo.toml

```diff
@@ -29,3 +33,6 @@ button = [
     "radix-leptos-icons/envelope-open",
     "radix-leptos-icons/reload",
 ]
+card = [
+    "dep:shadcn-ui-leptos-card"
```

Comment:
To use the button in the card examples, you need to include it here, see the Yew book example.

Reply:
Sorry, I could have been more explicit. All optional dependencies need to be included here, so `lucide-leptos` as well.

Reply:
right, that was dumb of me. fixed